edge-modules/TestAnalyzer/Controllers/DirectMethodStatusController.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestAnalyzer.Controllers
+{
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Mvc;
+
+    [Route("api/[controller]")]
+    [ApiController]
+    public class DirectMethodStatusController : Controller
+    {
+        // POST api/directmethodstatus
+        [HttpPost]
+        public async Task Post(ResponseStatus methodCallStatus)
```

Comment:
http://hamidmosalla.com/2017/03/29/asp-net-core-action-results-explained/; return Task<IActionResult> and use "return NoContent();" to return NoContentResult.
```suggestion
        public async Task<IActionResult> PostAsync(ResponseStatus methodCallStatus)
```

Reply:
Why not a status code of 200? The controller exposes a method for this, so we can return:
```
return this.StatusCode(200)
```